README.md

```diff
@@ -71,8 +93,11 @@ MSALInteractiveTokenParameters *interactiveParams = [[MSALInteractiveTokenParame
 The master branch has been copied over to main branch. The master branch will contain updates only until version 1.2.14, for further releases please refer to 'main' branch instead of 'master'.
```

Comment:
There's a typo in the title above this paragraph:

> Master branch **deprercation**

Reply:
Good spot, thanks @diegojerezba